api4/post_test.go

```diff
@@ -35,25 +34,11 @@ func TestCreatePost(t *testing.T) {
 	CheckNoError(t, resp)
 	CheckCreatedStatus(t, resp)
 
-	if rpost.Message != post.Message {
-		t.Fatal("message didn't match")
-	}
-
-	if rpost.Hashtags != "#hashtag" {
-		t.Fatal("hashtag didn't match")
-	}
-
-	if len(rpost.FileIds) != 0 {
-		t.Fatal("shouldn't have files")
-	}
-
-	if rpost.EditAt != 0 {
-		t.Fatal("newly created post shouldn't have EditAt set")
-	}
-
-	if rpost.Props[model.PROPS_ADD_CHANNEL_MEMBER] != nil {
-		t.Fatal("newly created post shouldn't have Props['add_channel_member'] set")
-	}
+	require.Equal(t, post.Message, rpost.Message, "message didn't match")
+	require.Equal(t, "#hashtag", rpost.Hashtags, "hashtag didn't match")
+	require.Equal(t, 0, len(rpost.FileIds), "shouldn't have files")
```

Comment:
You can use `require.Len` here. Please take a look at the checks where this happens (a `require.Equal` that contains a `len` call) as this change is applicable to all those.

Reply:
Thanks, this update has been made throughout.